contentcuration/contentcuration/utils/publish.py

```diff
@@ -799,4 +820,12 @@ def publish_channel(
             os.remove(kolibri_temp_db)
         channel.main_tree.publishing = False
         channel.main_tree.save()
+
+    elapsed = time.time() - start
+
+    if elapsed > PUBLISHING_UPDATE_THRESHOLD:
+        try:
+            raise SlowPublishError(elapsed, channel_id)
+        except SlowPublishError as e:
+            report_exception(e)
```

Comment:
Can this be simplified to `report_exception(SlowPublishError(elapsed, channel_id))`?

Reply:
Done

Reply:
Are there any caveats to this? Just curious if there was a reason why the try/except was used.

Reply:
@bjester I believe the `try/except` block makes it look more clear as to what we are doing there. I think going ahead with try/except would be a better idea IMO.

Reply:
I think we have used this pattern because of how sentry captures exceptions using `traceback` - if we haven't actually raised the exception, then it won't be accessible in the stack trace when Sentry tries to log it.

Reply:
Thanks all! I was thinking maybe the traceback wouldn't be available, but perhaps we don't need it for unique errors like this. So keeping the try/except, the only thing that's missing is a code comment IMO! I see something like that for the `SlowSyncError`: "# we need to raise it to get Python to fill out the stack trace."

Reply:
@ozer550 Sounds good. The only difference then is that your change is missing the code comment, which would answer my original question for any future devs on this specific code!

Reply:
@bjester I have reverted the changes to raise the exception and commented on it.